client/lib/analytics/index.js

```diff
@@ -125,6 +125,11 @@ var analytics = {
 		}
 	},
 
+	recordEvent: function( tracksEventName, tracksEventProperties, category, action, label, value ) {
```

Comment:
6 params is way too much :)
Isn't there any way to extract: `category`, `action`, `label` or `value` from `tracksEventName` and `tracksEventProperties` or the other way around?